setup.sh

```diff
@@ -583,6 +588,29 @@ function synchronize_files()
   say "$app_name installed into $HOME"
 }
 
+function install_nucompletion()
+{
+  local completions_file="${libdir}/kw-completions.nu"
+
+  if [[ -f "$completions_file" ]]; then
+    return
+  fi
+
+  if [[ $(ask_yN "Nushell detected. Want to download completions for kw?") =~ '0' ]]; then
+    say "If you change your mind, download it manually download from https://github.com/nushell/nu_scripts/tree/main/custom-completions/kw"
+    return
+  fi
+
+  curl --silent https://raw.githubusercontent.com/nushell/nu_scripts/main/custom-completions/kw/kw-completions.nu --output "${completions_file}"
+
+  if [[ "$?" != 0 ]]; then
+    complain 'Failed to download nu completions for kw. Try manually downloading from https://github.com/nushell/nu_scripts/tree/main/custom-completions/kw'
+    return
+  fi
+
+  say "Nu completions downloaded successfully, add \`source ${completions_file}\` to your nushell config file (\$nu.config-file)"
```

Comment:
Why don't you do it for your Nutshell user? Ask them if they want to add it now and handle it for them. Check how we handle it for .bashrc, we have a couple of functions that helping adding and removing this type of line.

Reply:
Even tho I could add the source to the config.nu during the installation, this may not be that simple. Nushell configuration files are simple as bash files, they have some sort of organization. Probably adding the source at the end of the file would work but mess with the users config organization.